resources/views/layouts/app.blade.php

```diff
@@ -163,7 +165,7 @@
                                                     @endif
 
                                                     @if ($web_notifications_info['notifications']->hasMorePages())
-                                                        <li class="web-notification-more">
+                                                        <li class="Container-flweb-notification-more">
```

Comment:
Have a space in between two css classes
Container-flweb-notification-more. -> container-fl web-notification-more